app/src/main/java/com/chess/clock/engine/TimeControlManager.java

```diff
@@ -172,17 +172,22 @@ public void prepareNewEditableTimeControl() {
         // Set default stage and time increment
         Stage stage = new Stage(0, 300000);
         TimeIncrement timeIncrement = new TimeIncrement(TimeIncrement.Type.FISCHER, 5000);
+        TimeControl blank = new TimeControl(null, new Stage[]{stage}, timeIncrement);
 
         // Set current editable time control with a new "blank" time control
-        mEditableTimeControl = new TimeControl(null, new Stage[]{stage}, timeIncrement);
+        try {
+            mEditableTimeControl = new TimeControlWrapper(blank, (TimeControl) blank.clone());
+        } catch(CloneNotSupportedException e) {
+            e.printStackTrace();
```

Comment:
Hhmm...I'm torn between if we want to just log or log and throw an `IllegalStateException` so that we know there is trouble here...

I understand it "impossible" for this to occur in theory, but I'm not very trusting :P

Reply:
Fixed in 0faf160